src/main/java/core/basesyntax/Excavator.java

```diff
@@ -0,0 +1,14 @@
+package core.basesyntax;
+
+public class Excavator extends Machine {
+    @Override
+    public void doWork() {
+        System.out.println("Excavator starts work");
+    }
+
+    @Override
+    public void stopWork() {
+        System.out.println("Excavator stops work");

```

Comment:
Similarly, the message 'Excavator stops work' should be more descriptive. Indicate the specific type of machine that has stopped working, such as 'Excavator has finished digging.'